hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -1849,8 +1849,7 @@ void testWriteReadWithTimestampWithoutTZ(HoodieTableType tableType) {
         .option(FlinkOptions.PRECOMBINE_FIELD, "f1")
         .option(FlinkOptions.TABLE_TYPE, tableType)
         .option(FlinkOptions.WRITE_UTC_TIMEZONE, false)
-        //FlinkOptions.READ_UTC_TIMEZONE doesn't affect in MergeOnReadInputFormat since the option isn't supported in AvroToRowDataConverters
-        //.option(FlinkOptions.READ_UTC_TIMEZONE, false)
+        .option(FlinkOptions.READ_UTC_TIMEZONE, true)
```

Comment:
The default value itself is true. Why do we need to set it explicitly in test?
Can you also write a test with assertions on the record itself?

Reply:
Can we also add a test case where `FlinkOptions.READ_UTC_TIMEZONE` is false.

Reply:
+1

Reply:
@danny0405 done. Please review and merge when possible.